process-automation/latest/model/authentication.md

```diff
@@ -43,3 +45,41 @@ To create authentication:
 The following is an authentication created using Basic authentication and a username and password.
 
 ![Authentication]({% link process-automation/images/authentication.png %})
+
+## Upgrade authentication for deployed application
+
+You can upgrade the authentication of your application using the same version of your application. You can also upgrade the authentication of your application and change the deployed version.
```

Comment:
i'm not sure that we need that sentences. authentication is strictly realted to the application version, if the new version of application has identical authentication as currently deployed one, you can skip authentication step, if the new version of app has NOT identical authentication (with different name, description, credentials etc etc) user will be forced to upgrade values for the new authentications

Reply:
Changed.